.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "spawner-bin"]
 	path = spawner-bin
 	url = https://github.com/klenin/spawner-bin.git
+[submodule "lib/cats-modules"]
+	path = lib/cats-modules
+	url = https://github.com/BabichMikhail/cats-modules
```

Comment:
Let's move repository before the merge.
See Settings->Transfer ownership in GitHub UI.